operations/enterprises.md

```diff
@@ -512,6 +519,12 @@ Returns all space blocks \(out of order blocks or house use blocks\) colliding w
 * `Updated` - space block updated within the interval.
 * `Created` - space block created within the interval.
 
+#### Space block extent
```

Comment:
This should be right after the table with request properties.

Reply:
@siroky Fixed, when the deploy is done, I will merge it.